src/lib/middleware/assetMiddleware.ts

```diff
@@ -1,7 +1,9 @@
 import path from "path"
 import fs from "fs"
 import chalk from "chalk"
-import { isProduction, CDN_URL } from "lib/environment"
+import { isProduction } from "lib/environment"
+
+const { CDN_URL } = process.env
```

Comment:
It's not clear to me why the CDN_URL no longer can be made available through the `environment` module, what gives?

Reply:
My thinking was that normal ENV variables could be extracted the usual way wheres the `environment` file is more related to the runtime environment. By importing there only to export so that they can be imported elsewhere (by importing `environment`) creates a bit of indirection and breaks common node patterns. That said `environment`, `process.env`... it seems like it could be a bit.